src/screens/Referral/ProcessingClaim.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components/native';
+import { baseColors, fontSizes } from 'utils/variables';
+import { BoldText, LightText } from 'components/Typography';
+import Spinner from 'components/Spinner';
+
+const Status = styled(BoldText)`
+  font-size: ${fontSizes.giant}px;
+  text-align: center;
+  margin-top: 35px
```

Comment:
`;` Here as well :(
You are reusing this styling a lot. Maybe you could create a dumb component and import it to all related screens rather than repeating it? :)

Reply:
hmm it's just 2 components 🤔

Reply:
Hm, miscalculated - sorry. So then it's okay :)